plutus-ledger-api/src/Plutus/V1/Ledger/Api.hs

```diff
@@ -233,7 +234,7 @@ evaluateScriptRestricting verbose cmdata budget p args = swap $ runWriter @LogOu
             UPLC.runCek
                 (toMachineParameters model)
                 (UPLC.restricting $ PLC.ExRestrictingBudget budget)
-                (verbose == Verbose)
+                (if verbose == Verbose then Emit else NoEmit)
```

Comment:
No, I think we do just want to give them a simple toggle for now. Especially in this case, where the emit-with-timestamps mode is never going to be something that you use in the real version (because you'll need to litter your program with traces, which you would never do in reality). So I think it's fine to keep it like this for now. Also generally we want to keep the interface fairly fixed, otherwise we can't change anything without breaking the ledger, which is quite painful.